packages/eslint-plugin/src/configs/react-config.js

```diff
@@ -28,6 +28,17 @@ module.exports = {
     '@griffel/hook-naming': 'error',
     '@griffel/no-shorthands': 'error',
     '@griffel/styles-file': 'error',
+    'no-restricted-globals': [
+      'error',
+      {
+        name: 'window',
+        message: 'Get a reference to `window` via context.',
```

Comment:
This is a pretty generic message and we might consider customizing this message for v8/9/0.